### PR TITLE
feat: adds time zone setting for JDBC

### PR DESCRIPTION
**Pull request by killme2008**
## What's Changed in this PR

Document for https://github.com/GreptimeTeam/greptimedb/issues/3887

Related PR https://github.com/GreptimeTeam/greptimedb/pull/4064

## Checklist

- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

**Review by tisonkun — APPROVED** [review]

File: docs/nightly/en/greptimecloud/integrations/mysql.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ Use following connect string for your JDBC client.
 jdbc:mysql://<host>:4002/<dbname>?user=<username>&password=<password>
 ```
 
+Read the [time zone setting](/user-guide/client-libraries/java#time-zone) for JDBC.
```
Comment (nicecui) [inline docs/nightly/en/greptimecloud/integrations/mysql.md:39]:
```suggestion
Read the [time zone setting](https://docs.greptime.com/user-guide/client-libraries/java#time-zone) for JDBC.
```

File: docs/nightly/zh/greptimecloud/integrations/mysql.md [hunk 2]
```diff
@@ -34,6 +34,8 @@ mysql --ssl -u <username> -p -h <host> -P 4002 -A <dbname>
 jdbc:mysql://<host>:4002/<dbname>?user=<username>&password=<password>
 ```
 
+ JDBC 的时区设置请阅读[这篇文档](/user-guide/client-libraries/java#时区)。
```
Comment (nicecui) [inline docs/nightly/zh/greptimecloud/integrations/mysql.md:37]:
```suggestion
 JDBC 的时区设置请阅读[这篇文档](https://docs.greptime.cn/user-guide/client-libraries/java#时区)。
```

File: docs/v0.8/en/greptimecloud/integrations/mysql.md [hunk 3]
```diff
@@ -36,6 +36,8 @@ Use following connect string for your JDBC client.
 jdbc:mysql://<host>:4002/<dbname>?user=<username>&password=<password>
 ```
 
+Read the [time zone setting](/user-guide/client-libraries/java#time-zone) for JDBC.
```
Comment (nicecui) [inline docs/v0.8/en/greptimecloud/integrations/mysql.md:39]:
```suggestion
Read the [time zone setting](https://docs.greptime.com/user-guide/client-libraries/java#time-zone) for JDBC.
```

File: docs/v0.8/zh/greptimecloud/integrations/mysql.md [hunk 4]
```diff
@@ -34,6 +34,8 @@ mysql --ssl -u <username> -p -h <host> -P 4002 -A <dbname>
 jdbc:mysql://<host>:4002/<dbname>?user=<username>&password=<password>
 ```
 
+ JDBC 的时区设置请阅读[这篇文档](/user-guide/client-libraries/java#时区)。
```
Comment (nicecui) [inline docs/v0.8/zh/greptimecloud/integrations/mysql.md:37]:
```suggestion
 JDBC 的时区设置请阅读[这篇文档](https://docs.greptime.cn/user-guide/client-libraries/java#时区)。
```

File: docs/nightly/en/greptimecloud/integrations/mysql.md (hunk 1 above)
Comment (tisonkun) [reply docs/nightly/en/greptimecloud/integrations/mysql.md]: Why we need to pin the latest version here? I suppose the same version should work?

Comment (nicecui) [reply docs/nightly/en/greptimecloud/integrations/mysql.md]: These documents are displayed at https://greptime.cloud, so the internal links will not work.

**Review by nicecui — APPROVED** [review]

File: docs/nightly/en/greptimecloud/integrations/mysql.md (hunk 1 above)
Comment (tisonkun) [reply docs/nightly/en/greptimecloud/integrations/mysql.md]: Got it.

**Pull request merged**